lib/encoding/mnemonic.ex

```diff
@@ -5,8 +5,8 @@ defmodule Elixium.Mnemonic do
   @leading_zeros_of_mnemonic 11
   @regex_chunk_from_entropy Regex.compile!(".{1,#{@leading_zeros_of_mnemonic}}")
   @regex_chunk_to_entropy Regex.compile!(".{1,#{@leading_zeros_for_mnemonic}}")
-
-  @words to_string(:code.priv_dir(:elixium_core)) <> "/words.txt"
+#to_string(:code.priv_dir(:elixium_core))
+  @words "priv/words.txt"
```

Comment:
This breaks release's and pulling the package from hex rather than a local installation

Reply:
Fixed & Working